<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_total_marks">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mark_pos_1_pushButt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mark_pos_2_pushButt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mark_neg_onethird_pushButt">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-1/3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mark_neg_twothird_pushButt">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-2/3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mark_pos_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mark_pos_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mark_neg_onethird">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mark_neg_twothird">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mark_pos_0_pushButt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mark_pos_0">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Marks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_correct_ans">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>101</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct Answer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>101</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Questions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_correct_ans_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>468</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Incorrect Answer</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>801</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_total"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_correct_ans"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_incorrect"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
